project_national/housing_characteristics/HVAC Heating Type.tsv

```diff
@@ -1,112 +1,112 @@
-2A - FL, GA, AL, MS	Fuel Oil	1970s	0	0.709153	0	0.290847	0	19	37784.4	1.46415e-05
-2A - FL, GA, AL, MS	Natural Gas	1970s	0	0.709153	0	0.290847	0	19	37784.4	0.000515461
-2A - FL, GA, AL, MS	None	1970s	0	0	0	0	1	19	119099	0.000126348
-2A - FL, GA, AL, MS	Other Fuel	1970s	0	0.709153	0	0.290847	0	19	37784.4	4.1457e-05
-2A - FL, GA, AL, MS	Propane	1970s	0	0.709153	0	0.290847	0	19	37784.4	0.000118232
-2A - FL, GA, AL, MS	Electricity	1980s	0.333833	0.544325	0	0.121842	0	181	1.15212e+06	0.0119333
-2A - FL, GA, AL, MS	Fuel Oil	1980s	0	0.882566	0	0.117434	0	11	25275.3	1.26718e-05
-2A - FL, GA, AL, MS	Natural Gas	1980s	0	0.882566	0	0.117434	0	11	25275.3	0.000480924
-2A - FL, GA, AL, MS	None	1980s	0	0	0	0	1	21	126636	0.000155018
-2A - FL, GA, AL, MS	Other Fuel	1980s	0	0.882566	0	0.117434	0	11	25275.3	4.44132e-05
-2A - FL, GA, AL, MS	Propane	1980s	0	0.882566	0	0.117434	0	11	25275.3	0.000121305
-2A - FL, GA, AL, MS	Electricity	1990s	0.43798	0.471951	0	0.0900688	0	163	1.08267e+06	0.0101132
-2A - FL, GA, AL, MS	Fuel Oil	1990s	0	0.816074	0	0.183926	0	12	27077.8	9.13849e-06
-2A - FL, GA, AL, MS	Natural Gas	1990s	0	0.816074	0	0.183926	0	12	27077.8	0.000625902
-2A - FL, GA, AL, MS	None	1990s	0	0	0	0	1	13	70500.8	0.000120226
-2A - FL, GA, AL, MS	Other Fuel	1990s	0	0.816074	0	0.183926	0	12	27077.8	4.25597e-05
-2A - FL, GA, AL, MS	Propane	1990s	0	0.816074	0	0.183926	0	12	27077.8	0.000115614
-2A - FL, GA, AL, MS	Electricity	2000s	0.42935	0.538306	0	0.0323444	0	168	495998	0.0117365
-2A - FL, GA, AL, MS	Fuel Oil	2000s	0	0.716221	0	0.283779	0	21	21769.1	8.15556e-06
-2A - FL, GA, AL, MS	Natural Gas	2000s	0	0.716221	0	0.283779	0	21	21769.1	0.000893373
-2A - FL, GA, AL, MS	None	2000s	0	0	0	0	1	17	54228.9	0.000132201
-2A - FL, GA, AL, MS	Other Fuel	2000s	0	0.716221	0	0.283779	0	21	21769.1	3.36626e-05
-2A - FL, GA, AL, MS	Propane	2000s	0	0.716221	0	0.283779	0	21	21769.1	0.000114437
-2A - FL, GA, AL, MS	Electricity	2010s	0.42935	0.538306	0	0.0323444	0	168	495998	0.00342367
-2A - FL, GA, AL, MS	Fuel Oil	2010s	0	0.716221	0	0.283779	0	21	21769.1	2.90345e-06
-2A - FL, GA, AL, MS	Natural Gas	2010s	0	0.716221	0	0.283779	0	21	21769.1	0.000258093
-2A - FL, GA, AL, MS	None	2010s	0	0	0	0	1	17	54228.9	3.83793e-05
-2A - FL, GA, AL, MS	Other Fuel	2010s	0	0.716221	0	0.283779	0	21	21769.1	1.57376e-05
-2A - FL, GA, AL, MS	Propane	2010s	0	0.716221	0	0.283779	0	21	21769.1	4.58919e-05
-2A - FL, GA, AL, MS	Electricity	<1940	0.435379	0.384268	0	0.180353	0	23	132806	0.0011869
-2A - FL, GA, AL, MS	Fuel Oil	<1940	0	0.584791	0	0.415209	0	5	8110.56	3.4184e-06
-2A - FL, GA, AL, MS	Natural Gas	<1940	0	0.584791	0	0.415209	0	5	8110.56	0.000205618
-2A - FL, GA, AL, MS	None	<1940	0	0	0	0	1	5	29083.5	2.65671e-05
-2A - FL, GA, AL, MS	Other Fuel	<1940	0	0.584791	0	0.415209	0	5	8110.56	1.4124e-05
-2A - FL, GA, AL, MS	Propane	<1940	0	0.584791	0	0.415209	0	5	8110.56	4.85526e-05
+1A	Electricity	1940s	0.732707	0	0	0.267293	0	4	5667.8	0.000435254
+1A	Fuel Oil	1940s	0.2	0.2	0.2	0.2	0.2	0	0	9.00776e-08
```

Comment:
@lixiliu , @rajeee : I missed these uniform distribution rows in the resstock-estimation PRs. Do we know what happened and why these were uniformly distributed? There are a handful of rows.